source/mus_species_module.f90

```diff
@@ -168,6 +170,14 @@ subroutine mus_load_species( me, conf, parent, minLevel, nFields, physics, &
         & write(logUnit(1),*)'Variable has wrong type!'
     end if
 
+    !get molecular weight
+    call aot_get_val( L       = conf,                                          &
+      &               thandle = spc_handle,                                    &
+      &               key     = 'lambda',                                   &
+      &               val     = me%lambda,                                  &
+      &               ErrCode = iError,                                        &
+      &               default = 0.25_rk )
```

Comment:
Please align the `&` at the end of the lines.

Reply:
BUG: Molecular weight and trt relaxation parameter are both stored in `me%lambda`